lib/src/store/picture_storage.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:get/get.dart';
+import 'package:sqflite/sqflite.dart';
+import 'dart:io';
+
+class PictureStorage {
+  static pictureInformationMap(String courseId, String label, String picturePath, String note) {
+    return {
+      'courseId': courseId,
+      'label': label,
+      'note': note,
+      'picturePath': picturePath,
+    };
+  }
+
+  static takePictureToStorage(String courseId, String picturePath) async {
+    Database pictureDB = Get.find<Database>();
```

Comment:
Suggest adding final to this variable.

```suggestion
    final pictureDB = Get.find<Database>();
```